CMakeLists.txt

```diff
@@ -200,3 +200,17 @@ if (BENCHMARK_ENABLE_TESTING)
   enable_testing()
   add_subdirectory(test)
 endif()
+
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
```

Comment:
This needs to be off by default, with a CMake option to turn it on. Ex:
```
option(BENCHMARK_BUILD_NUGET_PACKAGE "Build a nuget package for benchmark" OFF)
[...]
if (BENCHMARK_BUILD_NUGET_PACKAGE)
[...]
endif()
```

And I think that all of the output should be put under a `nuget` subdirectory, instead of polluting the top-level build directory with everything.